cpp/src/groupby/sort/group_tdigest.cu

```diff
@@ -535,10 +535,12 @@ struct get_scalar_minmax {
 
   __device__ thrust::tuple<double, double> operator()(size_type group_index)
   {
-    // note: .element<T>() is taking care of fixed-point conversions for us.
-    return {static_cast<double>(col.element<T>(group_offsets[group_index])),
-            static_cast<double>(
-              col.element<T>(group_offsets[group_index] + (group_valid_counts[group_index] - 1)))};
+    auto const valid_count = group_valid_counts[group_index];
+    return valid_count > 0
+             ? thrust::make_tuple(
+                 static_cast<double>(col.element<T>(group_offsets[group_index])),
+                 static_cast<double>(col.element<T>(group_offsets[group_index] + valid_count - 1)))
+             : thrust::make_tuple(0.0, 0.0);
```

Comment:
Is 0 appropriate for the min/max an empty group? Where/how will these values get used later on?

Reply:
@nvdbaranec

Reply:
It's fine.  An empty digest results in a null return value when queried later on.

Reply:
Thinking about this some more, this might actually be a problem. An empty digest in a group of otherwise useful digests would still propagate a min value of 0, which is probably wrong.  Let me think on this a bit.